config/fieldparams/mainnet.go

```diff
@@ -26,4 +26,6 @@ const (
 	SyncCommitteeAggregationBytesLength   = 16            // SyncCommitteeAggregationBytesLength defines the length of sync committee aggregate bytes.
 	SyncAggregateSyncCommitteeBytesLength = 64            // SyncAggregateSyncCommitteeBytesLength defines the length of sync committee bytes in a sync aggregate.
 	MaxWithdrawalsPerPayload              = 16            // MaxWithdrawalsPerPayloadLength defines the maximum number of withdrawals that can be included in a payload.
+	BlobSize                              = 131072        // defined to match blob.size in bazel ssz codegen
+	MaxBlobsPerBlock                      = 4             // MAX_BLOBS_PER_BLOCK
```

Comment:
I originally had `MaxBlobsPerBlock` under beacon config. Looking at this, defining in fieldparam is a better place than config. It is a preset and preset is closer to fieldparams than beacon config